src/Symfony/Component/Routing/Route.php

```diff
@@ -168,11 +166,9 @@ public function getHost()
      *
      * This method implements a fluent interface.
      *
-     * @param string $pattern The host pattern
-     *
      * @return $this
      */
-    public function setHost($pattern)
+    public function setHost(?string $pattern)
```

Comment:
This was previously undocumented, but we have to allow `null` here because of the way routes are read from config files. Since we immediately turn `null` into an empty string in the first line, I don't think we would gain much, if we would make the interface stricter.